powershell/cmdlets/class.ts

```diff
@@ -488,10 +565,13 @@ export class CmdletClass extends Class {
     });
 
     const $this = this;
-    this.add(new Method('EndProcessing', dotnet.Void, { access: Access.Protected, override: Modifier.Override, description: 'Performs clean-up after the command execution' })).add(function* () {
-      // gs01: remember what you were doing here to make it so these can be parallelized...
-      yield '';
-    });
+    this.add(new Method('EndProcessing', dotnet.Void, { access: Access.Protected, override: Modifier.Override, description: 'Performs clean-up after the command execution' })).add(
+      function* () {
+        // gs01: remember what you were doing here to make it so these can be parallelized...
+        if ($this.state.project.autoSwitchView) {
```

Comment:
Can we move it to the ProcessRecordAsync function instead of End function?

Reply:
we can't move it to the ProcessRecordAsync function. Because this function will be invoked many times when we pipes many objects to a cmdlet. While EndProcessing only runs once in this case.